pages/plants/[id].js

```diff
@@ -65,6 +65,20 @@ export default function PlantDetail({ onToggleFavorite, favorites, plants }) {
               imageSrc="/assets/WaterIcon.svg"
               info={plant.waterNeeds}
             />
+            <PlantCharacteristics
+              headline="Pet-Friendly"
+              imageAlt="Paw Icon"
+              imageSrc="/assets/PawIcon.svg"
+              info={plant.petFriendly === true ? "yes" : "no"}
+            />
+            <StyledLink href={`/categories/${plant.categorySlug}`}>
+            <PlantCharacteristics
```

Comment:
I am not sure if it is not better from a user perspective to add a visually more speaking difference. To decide this is up to you, but the user could benefit, maybe from another color, a hint or something else. Maybe nice o think about later. :)